doc/fern-renderer-devicons.txt

```diff
@@ -71,5 +72,11 @@ VARIABLE				*fern-renderer-devicons-variable*
 <
 	Default: "   "
 
+-----------------------------------------------------------------------------
+COLORS				*fern-renderer-devicons-colors*
+
+You can specify colors manually by just defining syntax for fern buffer.
+See examples at https://github.com/ryanoasis/vim-devicons/issues/158.
```

Comment:
Thanks for the doc. But the issue has too many comments so that I could not find which one is an example thus could you please specify one particular comment by URL?

Reply:
Done!